internal/v1/api.yaml

```diff
@@ -1745,6 +1745,8 @@ components:
           $ref: '#/components/schemas/Installer'
         cacerts:
           $ref: '#/components/schemas/CACertsCustomization'
+        aap:
+          $ref: '#/components/schemas/AAPCustomization'
```

Comment:
Could you use `oapi-codegen` extended fields to customize the Go type to `AAP` here? The generated `Aap` is awful :-)